src/clib/solve_rate_cool_g.F

```diff
@@ -709,8 +709,8 @@ subroutine solve_rate_cool_g(icool, d, e, u, v, w, de,
 
                if (tgas(i) .le. 1.01_DKIND*temstart .and.
      &              edot(i) .lt. 0._DKIND) 
-     &              edot(i) = tiny8
-	       if (abs(edot(i)) .lt. tiny8) edot(i) = tiny8
+     &              edot(i) = 0._DKIND
+	       if (abs(edot(i)) .lt. tiny8) edot(i) = 0._DKIND
```

Comment:
I thought the point of using tiny as a minimum was that the next line (717) divides by edot and we don't want that to generate a divide by zero.

Reply:
Yeah, totally agree, I'll revert this.